src/main/java/org/openmrs/addonindex/service/ElasticSearchIndex.java

```diff
@@ -191,6 +193,24 @@ public AddOnInfoAndVersions getByModulePackage(String modulePackage) throws IOEx
 				.map(sr -> sr.source)
 				.collect(Collectors.toList());
 	}
+
+	@Override
+	public Collection<AddOnInfoAndVersions> getRecentReleases(int resultSize) throws IOException {
```

Comment:
Returns a collection of type AddOnInfoAndVersions instead of AddOnInfoSummaryAndStats because I have displayed the Release date on the release page in the UI. The release date is not available in AddOnInfoSummaryAndStats. Do let me know if the release date is not needed in that page and I can remove it(my thinking was that the release page should have two key pieces of info for the user to be able to figure out if there has been a newer version of the module that they use been released. Either the module version or the release date can be indicative of a newer version. For ex: the user doesn't remember the exact module version but possibly remembers that the last time that they updated that module was sometime around last time but now they see that a new version had been released 4 days back.